packages/headless/src/api/commerce/unified-api/unified-api-error-response.ts

```diff
@@ -0,0 +1,43 @@
+import {DisconnectedError} from '../../../utils/errors';
+
+export interface UnifiedCommerceAPIError {
+  errorCode: number;
+  message: string;
+  ignored?: boolean;
```

Comment:
What does `ignored` do?

Reply:
It's an artefact of the duplication for errors definitions. It will go away when I refactor as you proposed in your other comment.

As for what it does, it's there because some errors are not really errors that an implementer need to know about/need to take action.

For example, if the end user spams the search button, there will be a lot of request that gets aborted and pollute the console. 

So some type of errors are flagged as ignored, and the logger will simply silence those.